[Build] Local NewPipeExtractor build inclusion fails in settings.gradle.kts

### Checklist

- [x] I am able to reproduce the bug with the latest version given here: [CLICK THIS LINK](https://github.com/TeamNewPipe/NewPipe/releases/latest).
- [x] I made sure that there are *no existing issues* - [open](https://github.com/TeamNewPipe/NewPipe/issues) or [closed](https://github.com/TeamNewPipe/NewPipe/issues?q=is%3Aissue+is%3Aclosed) - which I could contribute my information to.
- [x] I have read the [FAQ](https://newpipe.net/FAQ/) and my problem isn't listed.
- [x] I have taken the time to fill in all the required details. I understand that the bug report will be dismissed otherwise.
- [x] This issue contains only one bug.
- [x] I have read and understood the [contribution guidelines](https://github.com/TeamNewPipe/NewPipe/blob/dev/.github/CONTRIBUTING.md).

### Affected version

latest dev branch (Commits on Nov 2, 2025)

### Steps to reproduce the bug

Hi,

I encountered an issue when trying to include a local NewPipeExtractor build in `settings.gradle.kts`. 
The example in the commented section fails to work as expected, causing errors when uncommented.

After uncommenting the lines:

```kotlin
// includeBuild('../NewPipeExtractor') {
//     dependencySubstitution {
//         substitute module('com.github.TeamNewPipe:NewPipeExtractor') using project(':extractor')
//     }
// }
```

The build fails with the following error:

```
> allen@allen-Inspiron-3558:/media/allen/WinUbuShare/NewPipe_ubuntu/branch_build_error_into_NewPipeExtractor/NewPipe$ ./gradlew build
> e: file:///media/allen/WinUbuShare/NewPipe_ubuntu/branch_build_error_into_NewPipeExtractor/NewPipe/settings.gradle.kts:28:14: Too many characters in a character literal ''../NewPipeExtractor''
> e: file:///media/allen/WinUbuShare/NewPipe_ubuntu/branch_build_error_into_NewPipeExtractor/NewPipe/settings.gradle.kts:30:9: Function invocation 'substitute(...)' expected
> e: file:///media/allen/WinUbuShare/NewPipe_ubuntu/branch_build_error_into_NewPipeExtractor/NewPipe/settings.gradle.kts:30:9: No value passed for parameter 'substitutedDependency'
> e: file:///media/allen/WinUbuShare/NewPipe_ubuntu/branch_build_error_into_NewPipeExtractor/NewPipe/settings.gradle.kts:30:20: Unresolved reference. None of the following candidates is applicable because of receiver type mismatch: 
> public fun Dependencies.module(group: String?, name: String, version: String?): ExternalModuleDependency defined in org.gradle.kotlin.dsl
> public fun DependencyHandler.module(notation: Any, clientModuleConfiguration: ClientModuleScope.() -> Unit): ClientModule defined in org.gradle.kotlin.dsl
> public fun DependencyHandler.module(group: String, name: String, version: String? = ..., configuration: String? = ..., classifier: String? = ..., ext: String? = ...): ClientModule defined in org.gradle.kotlin.dsl
> public fun DependencyHandler.module(group: String, name: String, version: String? = ..., configuration: String? = ..., classifier: String? = ..., ext: String? = ..., clientModuleConfiguration: ClientModuleScope.() -> Unit): ClientModule defined in org.gradle.kotlin.dsl
> e: file:///media/allen/WinUbuShare/NewPipe_ubuntu/branch_build_error_into_NewPipeExtractor/NewPipe/settings.gradle.kts:30:27: Too many characters in a character literal ''com.github.TeamNewPipe:NewPipeExtractor''
> e: file:///media/allen/WinUbuShare/NewPipe_ubuntu/branch_build_error_into_NewPipeExtractor/NewPipe/settings.gradle.kts:30:84: The character literal does not conform to the expected type String!
> e: file:///media/allen/WinUbuShare/NewPipe_ubuntu/branch_build_error_into_NewPipeExtractor/NewPipe/settings.gradle.kts:30:84: Too many characters in a character literal '':extractor''
> 
> FAILURE: Build failed with an exception.
> 
> * Where:
> Settings file '/media/allen/WinUbuShare/NewPipe_ubuntu/branch_build_error_into_NewPipeExtractor/NewPipe/settings.gradle.kts' line: 28
> 
> * What went wrong:
> Script compilation errors:
> 
>   Line 28: includeBuild('../NewPipeExtractor') {
>                         ^ Too many characters in a character literal ''../NewPipeExtractor''
> 
>   Line 30:         substitute module('com.github.TeamNewPipe:NewPipeExtractor') using project(':extractor')
>                    ^ Function invocation 'substitute(...)' expected
> 
>   Line 30:         substitute module('com.github.TeamNewPipe:NewPipeExtractor') using project(':extractor')
>                    ^ No value passed for parameter 'substitutedDependency'
> 
>   Line 30:         substitute module('com.github.TeamNewPipe:NewPipeExtractor') using project(':extractor')
>                               ^ Unresolved reference. None of the following candidates is applicable because of receiver type mismatch: 
>                                   public fun Dependencies.module(group: String?, name: String, version: String?): ExternalModuleDependency defined in org.gradle.kotlin.dsl
>                                   public fun DependencyHandler.module(notation: Any, clientModuleConfiguration: ClientModuleScope.() -> Unit): ClientModule defined in org.gradle.kotlin.dsl
>                                   public fun DependencyHandler.module(group: String, name: String, version: String? = ..., configuration: String? = ..., classifier: String? = ..., ext: String? = ...): ClientModule defined in org.gradle.kotlin.dsl
>                                   public fun DependencyHandler.module(group: String, name: String, version: String? = ..., configuration: String? = ..., classifier: String? = ..., ext: String? = ..., clientModuleConfiguration: ClientModuleScope.() -> Unit): ClientModule defined in org.gradle.kotlin.dsl
> 
>   Line 30:         substitute module('com.github.TeamNewPipe:NewPipeExtractor') using project(':extractor')
>                                      ^ Too many characters in a character literal ''com.github.TeamNewPipe:NewPipeExtractor''
> 
>   Line 30:         substitute module('com.github.TeamNewPipe:NewPipeExtractor') using project(':extractor')
>                                                                                               ^ The character literal does not conform to the expected type String!
> 
>   Line 30:         substitute module('com.github.TeamNewPipe:NewPipeExtractor') using project(':extractor')
>                                                                                               ^ Too many characters in a character literal '':extractor''
> 
> 7 errors
> 
> * Try:
> > Run with --stacktrace option to get the stack trace.
> > Run with --info or --debug option to get more log output.
> > Run with --scan to get full insights.
> > Get more help at https://help.gradle.org.
> 
> BUILD FAILED in 3s
```
```

--- a/settings.gradle.kts
+++ b/settings.gradle.kts
@@ -25,8 +25,9 @@ include (":app")
 // We assume, that NewPipe and NewPipe Extractor have the same parent directory.
 // If this is not the case, please change the path in includeBuild().
 
-//includeBuild('../NewPipeExtractor') {
+//includeBuild("../NewPipeExtractor") {
 //    dependencySubstitution {
-//        substitute module('com.github.TeamNewPipe:NewPipeExtractor') using project(':extractor')
+//        substitute(module("com.github.TeamNewPipe:NewPipeExtractor"))
+//            .using(project(":extractor"))
 //    }
 //}

